src/utils/AudioPlayer.cpp

```diff
@@ -0,0 +1,83 @@
+#include "AudioPlayer.h"
+#include <QDebug>
+
+//#define USE_QSOUNDEFFECT
+#define USE_MEDIAPLAYER
```

Comment:
On OS X, QMediaPlayer fails (at least while other audio is active?). QSoundEffect seems to work better. I haven't tested against other platforms yet.

I think you mentioned having difficulties with both of them. Can you explain the issues?